build/utils/npm_test.go

```diff
@@ -168,6 +168,36 @@ func TestBundledDependenciesList(t *testing.T) {
 	}
 }
 
+// Only for npm v6.
+// Conflicts in peer dependencies.
```

Comment:
Let's add to the description some more information about what this test is actually checking.
We should make it as clear as possible for the random reader.